core/src/test/java/org/frankframework/pipes/XmlIfTest.java

```diff
@@ -92,14 +92,27 @@ void emptySessionKeyNullInputTest() throws Exception {
 	}
 
 	@Test
-	void invalidXPathExpressionTest() throws Exception {
+	void invalidXMLInputTest() throws Exception {
```

Comment:
This test didn't fail because of an invalid xpath expression but because of invalid input